packages/runtime-dom/src/modules/attrs.ts

```diff
@@ -36,7 +37,12 @@ export function patchAttr(
     if (value == null || (isBoolean && !includeBooleanAttr(value))) {
       el.removeAttribute(key)
     } else {
-      el.setAttribute(key, isBoolean ? '' : value)
+      // Symbols are explicitly stringified as the underlying library
```

Comment:
This bug only manifests when using `shallowMount` in Test Utils, which is implemented using `transfromVNodeArgs`. That function is behind a `__DEV__` flag: https://github.com/vuejs/vue-next/blob/b4eb7e3866d7dc722d93a48f4faae1696d4e7023/packages/runtime-core/src/vnode.ts#L481

So we might want to put this check behind the same `__DEV__` flag?